frontend/public/locales/en/status.json

```diff
@@ -0,0 +1,52 @@
+{
+  "page-title": "Status Checker",
+  "status-checker-heading": "What is the Status Checker?",
+  "status-checker-content": "The Status Checker is a tool that lets you check the status of your Canadian Dental Care Plan application.",
+  "online-status-heading": "Who can check their application status online?",
+  "online-status-content": "If you applied for the Canadian Dental Care Plan, you can check your status online.",
+  "terms-of-use": {
+    "summary": "Terms and Conditions of Use",
+    "heading": "Terms of Use for the Canadian Dental Care Plan Status Checker",
+    "legal-terms": "<strong>PLEASE READ CAREFULLY.</strong> We are glad you are using the Canadian Dental Care Plan Status Checker. We have put together some important terms and conditions for using this service (“Terms of Use” or “Terms”). Please read them carefully. They are important because they govern your use of the Status Checker. We encourage you to read these Terms carefully because they affect and may affect your legal rights and obligations. If there is a conflict between these terms and those of the www.Canada.ca, these Terms will take priority.",
```

Comment:
```suggestion
    "legal-terms": "<strong>PLEASE READ CAREFULLY.</strong> We are glad you are using the Canadian Dental Care Plan Status Checker. We have put together some important terms and conditions for using this service (\"Terms of Use\" or \"Terms\"). Please read them carefully. They are important because they govern your use of the Status Checker. We encourage you to read these Terms carefully because they affect and may affect your legal rights and obligations. If there is a conflict between these terms and those of the www.Canada.ca, these Terms will take priority.",
```